helpers/dateTime.js

```diff
@@ -384,22 +384,35 @@ export function monthNameAbbrev(m: number): string {
 }
 
 /**
- * Return difference between start and end dates, ignoring any time components
+ * Return difference between start and end dates (by default ignoring any time components)
+ * if returnFractionalDays is true, then use time components and return a fractional number of days (e.g. 1.5 for 36 hours)
+ * Note: It is highly recommended that if you have an ISO string (e.g. '2022-01-01') you send the string
+ * rather than trying to convert it to a Date object, as the Date object may be converted to local time, which may not be what you want.
  * Note: v2 uses a.moment(b).diff(moment().startOf('day'), 'days') instead
  * @author @jgclark
  * @tests in jest file
- * @param {Date} startDate
- * @param {Date} endDate
+ * @param {string|Date} startDate - if string, must be in ISO format (e.g. '2022-01-01')
+ * @param {string|Date} endDate - if string, must be in ISO format (e.g. '2022-01-01')
+ * @param {boolean} returnFractionalDays (default: false) - if true, return a fractional number of days (e.g. 1.5 for 36 hours)
+ * otherwise, it truncates the decimal part (e.g. 1 for 36 hours)
  * @return {number} - number of days between startDate and endDate (rounded to nearest integer)
```

Comment:
This now won't always be "rounded to nearest integer"

Reply:
ok